src/components/Person/Person.jsx

```diff
@@ -1 +1,26 @@
-// export const Person = ({ person }) => ();
+export const Person = ({ person }) => {
+  const { name, age = undefined, sex, isMarried, partnerName } = person;
+
+  const ageText =
+    age !== undefined ? <p className="Person__age">I am {age}</p> : null;
+
+  let partnerText;
+
+  if (isMarried) {
+    if (sex === 'm') {
+      partnerText = `${partnerName} is my wife`;
+    } else if (sex === 'f') {
+      partnerText = `${partnerName} is my husband`;
+    }
+  } else {
+    partnerText = 'I am not married';
+  }
```

Comment:
- Avoid nested `if/else` statements, and `else if` at all.
- 'm' and 'f' are magic values, create constants for them